rusk-recovery/src/state.rs

```diff
@@ -274,8 +274,15 @@ where
     let state_dir = state_dir.as_ref();
     let state_id_path = rusk_profile::to_rusk_state_id_path(state_dir);
 
+    let mut loaded = false;
     let (vm, old_commit_id) = match snapshot.base_state() {
-        Some(state) => load_state(state_dir, state),
+        Some(state) => {
+            let load_result = load_state(state_dir, state);
+            if load_result.is_ok() {
```

Comment:
No need for this, there is a `?` at the end of the match

Reply:
Ok, pushed new version. I did it in case someone changes the code underneath, so that it conveys the message, but ok